sdmetrics/visualization.py

```diff
@@ -259,3 +263,49 @@ def get_cardinality_plot(real_data, synthetic_data, child_table_name, parent_tab
     )
 
     return fig
+
+
+def get_column_plot(real_data, synthetic_data, column_name, plot_type=None):
+    """Return a plot of the real and synthetic data for a given column.
+
+    Args:
+        real_data (pandas.DataFrame):
+            The real table data.
+        synthetic_data (pandas.DataFrame):
+            The synthetic table data.
+        column_name (str):
+            The name of the column.
+        plot_type (str or None):
+            The plot to be used. Can choose between ``distplot``, ``bar`` or ``None``. If ``None`
+            select between ``distplot`` or ``bar`` depending on the data that the column contains,
+            ``distplot`` for datetime and numerical values and ``bar`` for categorical.
+            Defaults to ``None``.
+
+    Returns:
+        plotly.graph_objects._figure.Figure
+    """
+    if plot_type not in ['bar', 'distplot', None]:
+        raise ValueError(
+            f"Invalid plot_type '{plot_type}'. Please use one of ['bar', 'distplot', None]."
+        )
+
+    if column_name not in real_data.columns:
+        raise ValueError(f"Column '{column_name}' not found in real table data.")
+    if column_name not in synthetic_data.columns:
+        raise ValueError(f"Column '{column_name}' not found in synthetic table data.")
+
+    column_is_datetime = is_datetime(real_data[column_name])
+    real_column = real_data[column_name]
+    dtype = real_column.dropna().infer_objects().dtype.kind
```

Comment:
minor: I think `column_is_datetime` and `dtype` can be moved into the `if plot_type is None` block since they're only used in that case.